src/main/php/lang/ast/emit/PHP.class.php

```diff
@@ -988,6 +988,13 @@ protected function emitNewClass($result, $new) {
     array_shift($result->type);
   }
 
+  protected function emitCallable($result, $callable) {
+    $t= $result->temp();
+    $result->out->write('fn(...'.$t.') => ');
+    $this->emitOne($result, $callable->expression);
```

Comment:
Once the RFC merged to PHP 8.1, we should move this to a trait for PHP 7.4 and 8.0 and emit native code on PHP 8.1

Reply:
Now the only thing we need to do is to remove the `use CallablesAsClosures` from the `PHP81` emitter.